plugins/postgresql/postgresql_test.go

```diff
@@ -12,7 +13,7 @@ func TestPostgresqlGeneratesMetrics(t *testing.T) {
 	p := &Postgresql{
 		Servers: []*Server{
 			{
-				Address:   "sslmode=disable",
+				Address:   fmt.Sprintf("host=%s user=postgres sslmode=disable", testutil.GetLocalHost()),
```

Comment:
For the record, I had to change the user because the postgres docker image needs uses it to initialize the DB configuration. If I wanted to use some other user (like the default/session one) then I had to customize make a custom docker image which I didn't want to do.